reefperf/tests/test_files/test_node_types_config.json

```diff
@@ -1,24 +1,32 @@
 {
-    "app-node-types": {
-        "http": {
-            "driver-class": "TestCloudDriver",
-            "ram-size": "4GB",
-            "ssd-size": "4GB",
-            "username": "ubuntu"
-        },
-        "cache": {
-            "driver-class": "TestCloudDriver",
-            "ram-size": "8GB",
-            "ssd-size": "8GB",
-            "username": "ubuntu"
-        },
-        "database": {
-            "driver-class": "TestCloudDriver",
-            "ram-size": "16GB",
-            "sdd-size": "16GB",
-            "username": "ubuntu"
-        }
-    },
-    "test-nodes-types": {
-    }
+	"app_node_types": {
+		"http": {
+			"driver_class": "DummyCloudDriver",
+			"create_driver_parameters": {},
```

Comment:
maybe if it is empty then we don't need it here? Also, what would go there? Driver is a singleton, I think so it only gets created once per run anyway